library/src/helpers/parser.ts

```diff
@@ -1,27 +1,26 @@
-import { parse, parseFromUrl, registerSchemaParser } from '@asyncapi/parser';
-// @ts-ignore
-import openapiSchemaParser from '@asyncapi/openapi-schema-parser';
-// @ts-ignore
-import avroSchemaParser from '@asyncapi/avro-schema-parser';
+import { Parser as AsyncapiParser, fromURL } from '@asyncapi/parser';
+import {OpenAPISchemaParser} from '@asyncapi/openapi-schema-parser';
+import {AvroSchemaParser} from '@asyncapi/avro-schema-parser';
 // @ts-ignore
 import protoSchemaParser from '@asyncapi/protobuf-schema-parser';
 
 import { ErrorObject, ParserReturn, FetchingSchemaInterface } from '../types';
 
 import { VALIDATION_ERRORS_TYPE } from '../constants';
 
-registerSchemaParser(openapiSchemaParser);
-registerSchemaParser(avroSchemaParser);
-registerSchemaParser(protoSchemaParser);
+const asyncapiParser = new AsyncapiParser();
+asyncapiParser.registerSchemaParser(OpenAPISchemaParser());
+asyncapiParser.registerSchemaParser(AvroSchemaParser());
+//asyncapiParser.registerSchemaParser(protoSchemaParser);
```

Comment:
Until proto parser is updated to newest parser API it will be disabled here.

Reply:
Would be cool to leave that comment in the code.

Reply:
Should have it's own issue after this is merged 🙂

Reply:
Added a comment as well